providers/group.rb

```diff
@@ -101,7 +101,11 @@
 end
 
 def exists?
-  check = Mixlib::ShellOut.new("dsquery group -name \"#{new_resource.name}\"").run_command
+  cmd_user   = new_resource.cmd_user
+  cmd_pass   = new_resource.cmd_pass
+  cmd_domain = new_resource.cmd_domain
+  check = CmdHelper.shell_out("dsquery group -name \"#{new_resource.name}\"", cmd_user, cmd_pass, cmd_domain)
```

Comment:
To maintain consistency suggest:
`new_resource.cmd_user, new_resource.cmd_pass, new_resource.cmd_domain`

Reply:
I don't understand , should i put it in one line? i copied directly as it is in user.rb resource. What should my next step should be?

thanks,

Reply:
Yeah I was suggesting to place it on one line, but I see what you used, doesn't matter really.

Reply:
Thanks for accepting the pull request. I thanks for the cookbook.